docs/source/contributing.rst

```diff
@@ -36,28 +36,24 @@ Wagtailtrans made it very easy to setup a runnable Django project to help with t
 
 * We suggest you use a virtual environment for development.
 
-* Activate your virtual environment and install following packages.
+* Set PostgreSQL Database Authentication parameters host, user, and password as environment variables.
```

Comment:
This is a very good suggestion! However, In my opinion, this needs a bit more discussions on how should we implement split/local settings for `sandbox` application. Maybe a separate Issue?